### ISSUE TITLE
weird language constructs

### ISSUE THREAD

**Issue by SwissalpS** [minetest-mods/3d_armor#69]
https://github.com/minetest-mods/3d_armor/blob/b6651fd396dcd4ae6d75881cbf1712e5210eac9c/3d_armor/locale/3d_armor.es.tr#L61
This works well for helmet and chestplate, but with boots and pants it should be: "Your boots **are** almost broken"

In Spanish: "Tu**s** **b**otas **están** a punto de romperse"

In English it kinda works to use capitals here, but in Spanish and probably also in French, Portuguese and Italian, one would use lower-case. Probably some other languages too.

German is a language that capitalizes names, there it's correct. Here too, it would need to be: "Deine Stiefel **sind** fast kaputt".
However the rest use singular form **ist**.

Many languages have their own way, it kinda misses the point of translating when sentences sound so weird.

Suggestion: have a translator string for all items instead of using substitutions.

**Comment by tenplus1** [comment]
We could simplify it and maybe have "@1 almost broken!" or "@1 has broken!" so that you wouldn't need a translation for each piece of armor or even have to detect what has broken in order to display it.

**Comment by SwissalpS** [comment]
Not really.  It doesn't fix the problem even in English: "Sword has broken", yes but "Leggins/pants **has** broken", no, not proper English.

"Botas se han roto", meh. "Las botas se han roto", "Se han roto las botas", yes.
"Escudo se ha roto", can understand, but ugh. Much better: "El escudo se ha roto", "Se ha roto el escudo"  
"Pala se ha roto", sounds like a preschooler. Better: "La pala se ha roto", "Se ha roto la pala"  

"Pala casi se rompe" sounds weird. Better "La pala casi se rompe" or "La pala se romperá en seguida"
And "Botas casi se rompe" is just painful. "Botas casi se rompen" is only slightly better.

Languages with articles mostly need them as much as plurals affect the English language. It's even weirder than saying "Spade broken" instead of "The spade has broken".

Some tastes of German:
"Stiefel beinahe kaputt/gebrochen", "Spaten beinahe gebrochen";
"Stiefel gebrochen", "Spaten gebrochen"
Kinda works but they all sound weird.
"Die Stiefel sind bald kaputt", "Der Spaten ist beinahe gebrochen": sounds so much better.
"Die Stiefel sind zerstört", "Der Spaten ist zerstört", delightful :)

Shortening the phrase doesn't fix the issue in my view.
It's like shortening a maths equation from "1 + 1 = 2" to "1 1 2"
Yes, we can figure out what "1 1 2" might be saying, but it's distracting us and no maths
teacher would allow it.

In English we can mostly work around the plural problem with "The @1 has/have broken." But in some languages this might get really hard to read as there might be spaces involved and f/m options too.
"El/la/los/las @1 ...", "Die/der @1 ..."

French is notorios with how it adapts surrounding words:
"La pelle s'est cassé**e**" but "Le casque s'est cassé" and "Les bottes se **sont** cassé**es**"
